osh/hub/scripts/osh-retention

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Copyright contributors to the OpenScanHub project.
+
+"""
+Script for cron that performs the retention policy
+"""
+import datetime
+import logging
+import os
+import shutil
+from datetime import date
+
+logger = logging.getLogger(__name__)
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'osh.hub.settings'
+
+
+def delete_tasks_results(tasksToDelete):
+    from kobo.hub import models
+    for task in tasksToDelete:
+        try:
+            pathToDelete = models.Task.get_task_dir(task.id)
```

Comment:
The `tasksToDelete` identifier is not adhering to PEP8 variable naming guidelines.